### PR TITLE
add docker stats support

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @crosbymichael @icecrime :dancer:

File: api/utils.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func proxy(tlsConfig *tls.Config, addr string, w http.ResponseWriter, r *http.Re
 
 	copyHeader(w.Header(), resp.Header)
 	w.WriteHeader(resp.StatusCode)
-	io.Copy(w, resp.Body)
+	io.Copy(NewWriteFlusher(w), resp.Body)
```
Comment (aluzzardi) [inline api/utils.go]: Do we need to use the write flusher for every proxy?

Comment (vieux) [inline api/utils.go]: It doesn't hurt I believe, it's good improvement overall.

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**